README.md

```diff
@@ -1,143 +0,0 @@
-# Tokens² Curated List & Badges
-
-This demo demonstrates how to fetch all tokens with badges using only Metamask.
-Please run `yarn` followed by `yarn start`. The html must be served as Metamask won't interact with `file://`.
```

Comment:
This is overly complex for a simple example. There are tons of tools online to run snippets. No one wants to clone a repo for a demo.

Repl.it
CodeSandbox